### DISCUSSION TITLE
Question: How the angle works?

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#140] [Q&A] [answered]
It is a question that I have since version 0.5.0, How does the `angle` property work on objects? More specifically, how to know the exact number of angle to make a 360 turn, in some objects with an `angle = 40` they have already made more than one turn, maybe it is good to do some function for this, since I think it is somewhat confusing

**Comment by slmjkdbtl** [comment, marked as answer]
Angle is using radians instead of degrees. There's `rad2deg()` and `deg2rad()` but I think they're yet undocumented.
